docs/data/stashcache/install-origin.md

```diff
@@ -0,0 +1,104 @@
+# StashCache Origin Installation Guide
+
+This document describes how to install a StashCache origin service. The installation utilizes XRootD and HTCondor for file storage and monitoring, respectively. The role of the Origin and Cache is different, following explanation may help to decide when would you need to install which:
+
+* StashCache cache: role of **"cache"** server is to keep data cached and immediately available (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach) or CVMFS) within Stash federation (without re-transferring from "origin").
+* StashCache origin: The authoritative data server that hosts files locally and serves them to users upon transfer request (via [stashcp](https://support.opensciencegrid.org/support/solutions/articles/12000002775-transferring-data-with-stashcach) or CVMFS) unless data are already cached.
+
+---
+
+## Installation prerequisites for Origin
+
+Before starting the installation process, consider the following mandatory points:
+
+* __User IDs:__ If they do not exist already, the installation will create the Linux user IDs condor and xrootd
```

Comment:
What are the numeric UIDs?

Reply:
No idea.  Does that matter?

Reply:
I don't think so but it's information we try to provide in other installation docs